apiary-metastore-listener/README.md

```diff
@@ -38,7 +43,12 @@ The following shows an example JSON message representing a "CREATE_TABLE" event:
       "protocolVersion": "1.0",
       "eventType": "CREATE_TABLE",
       "dbName": "some_db",
-      "tableName": "some_table"
+      "tableName": "some_table",
+      "tableLocation": "s3://table_location",
+      "tableParameters": {
+          "hive.version": "2.3.0",
+          "hive.metastore.uris": "thrift://localhost:9083"
```

Comment:
You should amend the introductory line above to say something like "The following shows an example JSON message representing a "CREATE_TABLE" event when TABLE_PARAM_FILTER is set to `hive*`":

Reply:
done.